extensions/defaults/apiobjects.go

```diff
@@ -0,0 +1,7 @@
+package defaults
```

Comment:
One last nit Sam. Could put this in a different package, something more descriptive than just defaults?

Reply:
Fixed this.